log_outgoing_requests/validators.py

```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING, Any, Iterable, Optional
+
+if TYPE_CHECKING:  # pragma: nocover
+    from .models import OutgoingRequestsLogConfig
+
+
+def _validate_content_length(
+    content_length: Any, config: "OutgoingRequestsLogConfig"
+) -> bool:
+    max_content_length: Optional[int] = config.max_content_length
+    content_length_required: bool = config.content_length_required
+
+    # case 1: check of content_length not required
+    if not content_length_required:
+        return True
+
+    # case 2: no content-length in headers
+    if content_length is None:
+        return False
+
+    # normalize input
+    try:
+        content_length = int(content_length)
+    except (TypeError, ValueError):
+        return False
+
+    # case 3
+    return content_length <= max_content_length
+
+
+def _validate_content_type(
+    content_type: Optional[str], config: "OutgoingRequestsLogConfig"
+) -> bool:
+    # check 1 + 2: missing argument (or wrong type) for `content_type`
+    if content_type is None or not isinstance(content_type, str):
+        return False
+
+    allowed_mime_types: Iterable[str] = config.allowed_content_types or []
+    normalized = [item for string in allowed_mime_types for item in string.split(",")]
```

Comment:
I don't understand where these comma's are supposed to come from, can you explain that?